cfme/tests/physical_infrastructure/test_redfish_physical_server_details.py

```diff
@@ -2,16 +2,29 @@
 import pytest
 
 from cfme.physical.provider.redfish import RedfishProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
 
 pytestmark = [pytest.mark.provider([RedfishProvider], scope="function")]
 
 
 @pytest.fixture(scope="function")
 def physical_server(appliance, provider, setup_provider_funcscope):
-    # Get and return the first physical server
+    """Get and return the first physical server."""
     yield appliance.collections.redfish_physical_servers.all(provider)[0]
```

Comment:
Consider wrapping this so that the test can cleanly skip when there is no physical server available on the provider.

Otherwise the test ends in an ERROR state with IndexError.